clothcalc_placeholder.js

```diff
@@ -7765,10 +7766,11 @@
 
                             save["ItemUse.doIt"] = ItemUse.doIt;
                             try {
-                                var str = ItemUse.doIt.toString();
-                                var pos = str.indexOf("EventHandler.signal('item_used'");
-                                var inject = str.substr(0, pos) + "ItemUse.twdb(itemId,res);" + str.substr(pos);
-                                eval("ItemUse.doIt = " + inject);
+                                var toolkit = ItemUse.doItOrigin ? 'doItOrigin' : 'doIt',
+                                str = ItemUse[toolkit].toString(),
```

Comment:
Just to be sure, will these still be local declarations?

Reply:
Yes